### ISSUE TITLE
Consider changing public share and aggregation parameter encoding to match Poplar1 in draft-irtf-cfrg-vdaf-12

### ISSUE THREAD

**Issue by divergentdave** [jimouris/draft-mouris-cfrg-mastic#98]
We revised the encoding format of Poplar1 public shares and aggregation parameters in draft 12 of VDAF, in a way that's hopefully more intuitive and ergonomic. The key idea is that we no longer use arbitrary precision integers as an intermediate step when packing control bit correction words or candidate prefixes. Since Mastic's spec is still in flux, it might make sense to follow those changes here.

**Comment by cjpatton** [comment]
Yes indeed! Added to https://github.com/divviup/libprio-rs/issues/947.

**Comment by cjpatton** [comment]
It looks like we're already encoding the agg param and public share in the same way.
